07/sibenice/sibenice_funkce.py

```diff
@@ -7,8 +7,6 @@ def je_pismeno_ve_slove(pismeno, hadane_slovo):
     '''Zkontroluje, zda je zvolene pismeno v hadanem slove.'''
     if pismeno in hadane_slovo:
```

Comment:
Funkce bude vracet `None`, ne `False`, pokud pismeno ve slově nebude. Dal bych jen: `return pismeno in hledane_slovo`